managed/services/checks/checks_test.go

```diff
@@ -534,11 +534,11 @@ func TestFilterChecks(t *testing.T) {
 		},
 	}
 
-	checks := append(valid, invalid...)
+	checks := append(valid, invalid...) //nolint:gocritic
 
 	partiallyValidAdvisor := invalid[1]
 	partiallyValidAdvisor.Checks = partiallyValidAdvisor.Checks[0:1] // remove invalid check
-	expected := append(valid, partiallyValidAdvisor)
+	expected := append(valid, partiallyValidAdvisor)                 //nolint:gocritic
```

Comment:
let's replace `tab` by `space`

Reply:
I'm afraid it aligns with the comment above it, line 540.

Reply:
ah, ok